crates/opencascade-sys/build.rs

```diff
@@ -54,3 +64,65 @@ fn main() {
     println!("cargo:rerun-if-changed=src/lib.rs");
     println!("cargo:rerun-if-changed=include/wrapper.hxx");
 }
+
+struct OcctConfig {
+    include_dir: std::path::PathBuf,
+    library_dir: std::path::PathBuf,
+    is_dynamic: bool,
+}
+
+impl Default for OcctConfig {
+    #[cfg(feature = "builtin")]
+    fn default() -> Self {
+        let include_dir = occt_sys::occt_include_path();
+        let library_dir = occt_sys::occt_lib_path();
+
+        Self { include_dir, library_dir, is_dynamic: false }
+    }
```

Comment:
Optional: it could be nice to check `OCCT_VERSION` also against the bundled version so that we don't forget to update other if we update of these.

Reply:
I thought about it too but for now I haven't any ideas how to implement it.